feature/experimental/p4rt/ate_tests/lldp_packetout_test/lldp_packetout_test.go

```diff
@@ -0,0 +1,450 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lldp_packetout_test
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/internal/p4rtutils"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+	p4v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+const (
+	ipv4PrefixLen = 30
+	packetCount   = 100
+)
+
+var (
+	p4InfoFile                       = flag.String("p4info_file_location", "../../wbb.p4info.pb.txt", "Path to the p4info file.")
+	p4rtNodeName                     = flag.String("p4rt_node_name", "0/1/CPU0-NPU1", "component name for P4RT Node")
```

Comment:
is it possible to make it via a dict/map? E.g. if it is spitfire, then the name is what listed here.
Also, right now it is hardcoded. What if in our lab LC5 is used? Or what if a port connected to NPU3 is selected in Ondatra? 
With Q200 the mapping between ports and NPUs is pretty simple, maybe we can use that somehow to dynamically write this string?